src/pages/Dashboard/Dashboard.tsx

```diff
@@ -120,42 +127,35 @@ export const Dashboard = memo(() => {
     [appIsMobile, mobileTabs],
   )
 
-  const slideRenderer = (props: SlideRenderProps) => {
+  const slideRenderer = useCallback((props: SlideRenderProps) => {
     const { index, key } = props
     let content
-    switch (mod(index, 3)) {
+    const tab = mod(index, 3)
+    switch (tab) {
       case MobileTab.Overview:
         content = (
           <>
-            <Route exact path={`${path}`}>
-              <WalletDashboard />
-            </Route>
-            <Route path={`${path}/accounts`}>
-              <Accounts />
-            </Route>
+            <Route path='' element={walletDashboard} />
```

Comment:
q: is this empty path wanted?

Reply:
Yesser -  used wildcard in e3a5734924 to make it more explicit